tests/unit/helpers.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+class DummyExec:
+    def wait_output(self):
```

Comment:
Interesting mock.

Reply:
Is there a better way of doing it? I did it half asleep so wasn't sure.

Reply:
I think that's the way we should mock the container exec call. Maybe you can just move it to the test file, if it's being used in only one test file.